content/cpp/concepts/inheritance/terms/multiple-inheritance/multiple-inheritance.md

```diff
@@ -0,0 +1,133 @@
+---
+Title: 'Multiple Inheritance'
+Description: 'Enables classes to inherit characteristics from more than one class.'
```

Comment:
This can be rewritten like this to reduce ambiguity errors -

```suggestion
Description: 'Allows classes to inherit characteristics from more than one base class.'
```